cohorts/cohort.py

```diff
@@ -369,6 +377,7 @@ def load_dataframe(self, df_loader_name):
         Instead of joining a DataFrameJoiner with the Cohort in `as_dataframe`, sometimes
         we may want to just directly load a particular DataFrame.
         """
+        logger.debug('loading dataframe: {}'.format(df_loader_name))
```

Comment:
My attempted solution to endless " vs. ' debates is to at least try to preserve consistency within files; so if another file is mostly ' vs. " I'm fine with leaving that convention.